lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_marriage_and_civil_partnership.govspeak.erb

```diff
@@ -34,12 +34,14 @@
                locals: { calculator: calculator } %>
 
   <% elsif calculator.ceremony_country == 'vietnam' %>
-    You’ll need to download and complete the following documents:
-
-    <%= render partial: "appointment_documents_in_vietnam.govspeak.erb" %>
+    <% unless calculator.partner_is_national_of_ceremony_country?  %>
+      You’ll need to download and complete the following documents:
+    <% end %>
 
-    The embassy will confirm if your marriage can be registered and give you a link to book a ‘notice of marriage’ appointment online and a reference number. You must quote the reference number when booking your appointment.
+    <%= render partial: "appointment_documents_in_vietnam.govspeak.erb",
```

Comment:
@ikennaokpala Shouldn't the partial `appointment_documents_in_vietnam.govspeak.erb` be added to the `unless` statement?